app/os_android.go

```diff
@@ -131,6 +139,8 @@ import (
 	"gioui.org/unit"
 )
 
+const foregroundService = "org/gioui/GioForegroundService"
```

Comment:
This constant is only used once. I suggest folding the constant into the code using it.